### DISCUSSION TITLE
ControlNet Segmentation working as expected

### DISCUSSION THREAD

**Discussion by alexknows** [Mikubill/sd-webui-controlnet#2930] [Q&A]
I recently came across [OlivioTutorials](https://www.youtube.com/watch?v=8wpn7UO3FSk) video on ControlNet's segmentation, but it seems that the generated images is not adhering to it. 

Steps:

1. I generated the map in the ControlNet's section. 
2. Download map and make a few small tweaks in Photoshop. (e.g to remove a piece of wall and an object)
3. I replaced the original with the map. 
4. I added the prompts and adjusted all the settings accordingly. 

The image it generates has no similarity what so ever to the map I provided in ControlNet.

It seems as if ControlNet is not even enabled (but it is). No errors are thrown. 

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/6269334/3906fe10-97d0-446b-8cc9-f7075c358d2e)

**Comment by huchenlei** [comment]
Please update your controlnet extension. 1.1.313 is a very old version.

**Reply by alexknows** [reply]
Thanks for the answer @huchenlei 

I updated to the latest version, but still the results are way off. 

Am I missing something?

<img width="1836" alt="Screenshot 2024-05-24 at 6 46 35 PM" src="https://github.com/Mikubill/sd-webui-controlnet/assets/6269334/d3eeb723-78fe-4f93-8ba4-bf19e9a9cf3a">
